File: OSCommon/src/org/optimizationservices/oscommon/datastructure/osquery/StandardQuery.java
/**
 * @(#)StandardQuery 1.0 03/14/2004
 *
 * Copyright (c) 2004
 */
package org.optimizationservices.oscommon.datastructure.osquery;

import org.optimizationservices.oscommon.datastructure.osentity.OptimizationInfo;



/**
 *
 * <p>A data structure class that corresponds to an xml element in the OSqL schema.  
 *  
 * </p>
 * @author Robert Fourer, Jun Ma, Kipp Martin
 * @version 1.0, 03/14/2004
 * @since OS 1.0
*/
public class StandardQuery{

	public String currentState = null;

	public String serviceType = "solver";

	public OptimizationInfo optimization = null;
	
}//class StandardQuery

File: OSCommon/org/optimizationservices/oscommon/datastructure/osquery/StandardQuery.cs
using System;

using org.optimizationservices.oscommon.datastructure.osentity;

namespace org.optimizationservices.oscommon.datastructure.osquery{
	/// <summary>
	/// A data structure class that corresponds to an xml element in the OSqL schema. 
	/// @author Jun Ma	
	/// @version 1.0, 09/01/2005
	/// @since OS 1.0
	/// @copyright (c) 2005
	/// </summary>
	public class StandardQuery{

		public string currentState = null;

		public string serviceType = "solver";
	
		public OptimizationInfo optimization = null;

	}//class StandardQuery
}//namespace
